ros/noetic/tf2/00-allow-tf-repeated.patch

```diff
@@ -0,0 +1,45 @@
+diff --git a/tf2/src/cache.cpp b/tf2/src/cache.cpp
+index f9614c7..3994ab2 100644
+--- a/tf2/src/cache.cpp
++++ b/tf2/src/cache.cpp
```

Comment:
https://travis-ci.com/github/seqsense/aports-ros-experimental/jobs/386923791#L1523
This path seems be relative from geometry2 metapackage. `tf2/` should be removed.